### ISSUE TITLE
Migrate wiki to main repo

### ISSUE THREAD

**Issue by ringods** [twigjs/twig.js#730]
On one hand, Github wikis are a great thing. But on the other hand, one of it's drawbacks is that only repo maintainers have update access to the wiki.

In open source, contributions should also work for documentation. As a result, my request would be that one of the maintainers migrate all the wiki content into the main repo, e.g. in a `docs` folder.

Once the documentation is there, contributions to this documentation would follow the usual Pull Request pattern.

**Comment by RobLoach** [comment]
I'd be down. We could even publish rendered docs over with githubpages at http://twigjs.github.io/twig.js/ .

**Comment by ajiho** [comment]
Is there any progress on this issue? Perhaps I can help with this. I am familiar with using vitepress to generate beautiful documents, but my native language is not English. However, I can help generate a Chinese translation template so that you can transfer it to English yourself when the time comes?

**Comment by willrowe** [comment]
@ajiho contributions are welcomed. I think the plan is to have it published to github pages.

**Comment by ajiho** [comment]
> @ajiho contributions are welcomed. I think the plan is to have it published to github pages.
Of course, the deployment method can be chosen freely. I just want to get your opinion on whether to choose vitepress. I want to get your approval because there are too many tools for generating static documents. However, I personally highly recommend vitepress. If you want to see the document effect, you can check it out https://vitepress.dev/

**Comment by ajiho** [comment]
@willrowe Of course, the deployment method can be chosen freely. I just want to get your opinion on whether to choose vitepress. I want to get your approval because there are too many tools for generating static documents. However, I personally highly recommend vitepress. If you want to see the document effect, you can check it out https://vitepress.dev/

**Comment by willrowe** [comment]
@ajiho I'd prefer vitepress since I'm familiar with Vue and Vite. Any thoughts @RobLoach?

**Comment by RobLoach** [comment]
I worry about spending more time on making/maintaining a docs site than writing the actual docs. The less code/dependencies the better.

**Comment by ajiho** [comment]
@RobLoach Don't worry about this, when using viteperss, you just need to focus on writing MD files. A good document can add points to the project, as I have reviewed previous issues and found that someone has already complained about the current document

**Comment by RobLoach** [comment]
Sounds good with me 😁

**Comment by willrowe** [comment]
@ajiho that's great that it will run on MD files. I think you're good to go ahead then.

**Comment by ajiho** [comment]
@willrowe @RobLoach   

I took some time and completed it 

see: [#898](https://github.com/twigjs/twig.js/pull/898)
